lib/events/playback.go

```diff
@@ -107,6 +102,25 @@ func Export(ctx context.Context, rs io.ReadSeeker, w io.Writer, exportFormat str
 				if err != nil {
 					return trace.ConvertSystemError(err)
 				}
+			case teleport.YAML:
+				_, err := fmt.Println("---")
+				if err != nil {
+					return trace.ConvertSystemError(err)
+				}
+				err = utils.WriteYAML(w, event)
+				if err != nil {
+					return trace.ConvertSystemError(err)
+				}
+			case teleport.Text:
+				printEvent, ok := event.(*apievents.SessionPrint)
+				if !ok {
+					continue
+				}
+				// write bytes to writer
+				_, err := w.Write(printEvent.Data)
+				if err != nil {
```

Comment:
nit: smallest scope principle.

```suggestion
				if _, err := w.Write(printEvent.Data); err != nil {
```